pages/onboarding/index.tsx

```diff
@@ -59,7 +59,7 @@ const OnboardingPage: NextPage<Props> = ({ user }: Props) => {
       router.push('/directory');
     } catch (error) {
       setIsLoading(false);
-      console.error(error);
+      alert(error);
```

Comment:
`alert()` doesn't create a very good user experience and should be avoided in most cases.

Since we're using MUI, a toast alert is pretty easy to do. It seems that MUI calls them [Snackbars](https://mui.com/material-ui/react-snackbar/).